birdhouse/pavics-compose.sh

```diff
@@ -129,7 +129,7 @@ do
 
     # run postgres post-startup setup script
     # Note: this must run before the post-docker-compose-up scripts since some may expect postgres databases to exist
-    postgres_id=$(PROXY_SECURE_PORT=443 HOSTNAME=${PAVICS_FQDN} docker-compose ${COMPOSE_CONF_LIST} ps -q postgres)
+    postgres_id=$(PROXY_SECURE_PORT=443 HOSTNAME=${PAVICS_FQDN} docker-compose ${COMPOSE_CONF_LIST} ps -q postgres 2> /dev/null)
```

Comment:
Just curious. Which error/output is getting muted here?

Reply:
The error says that there is no container named postgres running (none of the default components require postgres now)

Reply:
I see! I suspected that was the case for a while.